kuka_sunrise_driver/src/kuka_sunrise/configuration_manager.cpp

```diff
@@ -404,23 +404,17 @@ bool ConfigurationManager::setCommandMode(const std::string & control_mode)
     RCLCPP_ERROR(robot_manager_node_->get_logger(), "Invalid control mode");
     return false;
   }
-  auto future_result = command_mode_client_->async_send_request(request);
```

Comment:
I don't know where to put it, but we should refactor parameter handling in this source file, too. Right?

Reply:
Yes, that should be done, but I would do it in a separate PR, as the current ParameterSetAccessRights structure of the base node is not sufficient for this purpose:
- in this case, the ConfigurationManager is instantiated while the node is in configuring (transition) state, but currently only primary states are supported in that struct
- either the struct should be modified to enable transition states or the code restructured -> FSM restructuring should be planned before